src/js/game/LevelMVC/LevelView.js

```diff
@@ -1159,6 +1159,62 @@ module.exports = class LevelView {
     }
   }
 
+  refreshActionPlane(levelData, indices) {
+    var sprite,
+      blockType;
+
+    for (var index in indices) {
+      if (indices[index] >= 0) {
+        let workingIndex = indices[index];
+
+        sprite = this.actionPlaneBlocks[workingIndex];
+        this.actionPlane.remove(sprite);
+
+        let y = Math.floor(workingIndex / this.controller.levelModel.planeHeight);
+        let x = workingIndex - (y * this.controller.levelModel.planeHeight);
```

Comment:
I could. I generally don't consider a bit of math a big deal. If it doesn't exist, it seems silly to not have an indexToCoordinates, provided we have a coordinatesToIndex. Would you like me to put that in?